Framework/Algorithms/inc/MantidAlgorithms/BinaryOperation.h

```diff
@@ -111,8 +111,8 @@ class DLLExport BinaryOperation : public API::Algorithm {
   /// masked. If so then the output spectra has zeroed data
```

Comment:
I think you are relying on `SpectrumInfo` being forward declared or included in one of the included headers. Not sure that is wise.